packages/react-mutation-mapper/src/store/DefaultMutationMapperStore.ts

```diff
@@ -2,11 +2,12 @@ import autobind from 'autobind-decorator';
 import { IOncoKbData, remoteData } from 'cbioportal-frontend-commons';
 import {
     genomicLocationString,
-    getMutationsToTranscriptId,
     groupMutationsByProteinStartPos,
     uniqueGenomicLocations,
+    getMutationsByTranscriptId,
```

Comment:
let's move this back between `genomicLocationString` and `groupMutationsByProteinStartPos`